docs/tutorials/migrate-from-neo4j.md

```diff
@@ -0,0 +1,497 @@
+    `CONTAINER ID` with `docker ps`, then run:
+
+    ```
+    docker cp /path_to_local_folder/shipping.csv <container_ID>:/usr/lib/memgraph/shipping.csv
+    ```
+
+    If the container ID is `bed1e5c9192d` and the file is locally located at
+    `C:/Data` the command would look like this:
+
+    ```
+    docker cp C:/Data/shipping.csv bed1:/usr/lib/memgraph/shipping.csv
+    ```
+
+3. To check if the files are inside the container, first run: 
+
+    ```
+    docker exec -it CONTAINER_ID bash
+    ```
+
+4. List the files inside the `/usr/lib/memgraph`. 
+
+    ```
+    C:\Users\Vlasta>docker ps
+    CONTAINER ID   IMAGE                        COMMAND                  CREATED         STATUS         PORTS                                                                    NAMES
+    bed1e5c9192d   memgraph/memgraph-platform   "/bin/sh -c '/usr/bi…"   2 minutes ago   Up 2 minutes   0.0.0.0:3000->3000/tcp, 0.0.0.0:7444->7444/tcp, 0.0.0.0:7687->7687/tcp   recursing_blackburn
+
+    C:\Users\Vlasta>docker cp C:/Data/shipping.csv bed1:/usr/lib/memgraph/shipping.csv
+
+    C:\Users\Vlasta>docker exec -it bed1 bash
+    root@bed1e5c9192d:/# ls /usr/lib/memgraph
+    auth_module  memgraph  python_support  query_modules  shipping.csv
+    root@bed1e5c9192d:/#
+    ```
+
+### 2. Gaining speed with indexes and analytical storage mode
+
+Although the dataset imported in this tutorial is quite small, one day you might
+want to import really big datasets with billions of nodes and relationships and
+you will require all the extra speed you can get. 
+
+To gain speed you can [create indexes](/reference-guide/indexing.md) on the
+properties used to connect nodes with relationships which are the values in the
+`_id` column in the CSV files, and in Memgraph they will be named `nodeID`.
+
+**To create indexes, run: **
+
+```cypher
+CREATE INDEX ON :Employee(nodeID);
+CREATE INDEX ON :Order(nodeID);
+CREATE INDEX ON :Product(nodeID);
+```
+
+You can also change the [storage mode](/reference-guide/storage-modes.md) from
+`IN_MEMORY_TRANSACTIONAL` to `IN_MEMORY_ANALYTICAL`. This will disable the
+creation of durability files (snapshots and WAL files) and you will no longer
+have any ACID guarantees. Other transactions will be able to see the changes of
+ongoing transactions. Also, transaction will be able to see the changes they are
+doing. This means that the transactions can be committed in random orders, and
+the updates to the data, in the end, might not be correct. 
+
+But, if you import on one thread, batch of data after a batch of data, there
+should be absolutely no issues, and you will gain 6 times faster import with 6
+times less memory consumption.
+
+After import you can switch back to the `IN_MEMORY_TRANSACTIONAL` storage mode or
+continue running analytics queries (only read queries) in the
+`IN_MEMORY_ANALYTICAL` mode to continue benefiting from low memory consumption. 
+
+**To switch between modes, run the following queries on a running instance:**
```

Comment:
Why is this bolded?

Reply:
Answer for all "Why is this bolded?" questions:

With indexes and storage mode it's because there is a lot of theory there as well, so if you just want to DO instead of know WHY, I'm bolding the parts that need to be done

For the rest - the heading h4 is horrible and easily skipped so I wanted to bold important queries necessary to create data